src/sentry/api/endpoints/organization_user_rollback.py

```diff
@@ -40,7 +40,10 @@ def get(self, request: Request, organization: Organization) -> Response:
 
         # Validate that the organization has rollbacks enabled
         if not organization.get_option("sentry:rollback_enabled", ROLLBACK_ENABLED_DEFAULT):
-            return Response(status=status.HTTP_404_NOT_FOUND)
+            return Response(
+                status=status.HTTP_403_FORBIDDEN,
```

Comment:
not that big of a deal, but I think it makes more sense to return a 404 here since the user does have access, the rollback just isn't enabled.